src/components/molecules/MapPartygoerOverlay/MapPartygoerOverlay.tsx

```diff
@@ -43,6 +43,7 @@ export const MapPartygoerOverlay: React.FC<MapPartygoerOverlayProps> = ({
       top: `${avatarHeight * (currentRow - 1)}%`,
       left: `${avatarWidth * (currentCol - 1)}%`,
       justifyContent: "center",
+      zIndex: "11",
```

Comment:
What is this needed for?

Reply:
That's because of the way party map components are layered. Cells, rooms are absolutely positioned with explicit zIndices, so we need it for user avatars as well